packages/create-plugin/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,51 @@
+# Windows Troubleshooting
+
+Here are some things to try and check if you experience problems while working with create-plugin or sign-plugin in Windows.
+
+## I am getting: `Unsupported operating system 'Windows' detected. Please use WSL with create-plugin`
```

Comment:
Instead of having this as a question, I would just have it as the opening statement.

Reply:
I agree with the points here and later but I intentionally left it that way (repetitive and a bit over-verbose) because developers looking for solutions won't read this page top to bottom but will scan through and expect to see their error message and their solution